prov/verbs/src/fi_verbs.c

```diff
@@ -3429,5 +3434,10 @@ static void fi_ibv_fini(void)
 
 VERBS_INI
 {
+	if (fi_ibv_init_verbs_info()) {
+		FI_INFO(&fi_ibv_prov, FI_LOG_CORE, "Unable to initialize verbs_info\n");
+		return NULL;
+	}
```

Comment:
library initialization routines are limited by what they can do/call.  For example, with static linking, this will run before the app has been fully initialized.  This is a heavy routine that invokes calls in lower-level libraries. We want to avoid performing these tasks in the INI function.